templates/patient/partials/delete.html.twig

```diff
@@ -9,7 +9,7 @@ The Patient Portal card for the Medical Record Dashboard
 {% block delete %}
 {% if isAdmin == true and allowPatientDelete == true %}
     <a class="btn btn-block btn-danger text-light deleter delete"
-        href="{{ urlWebRoot|attr }}/interface/patient_file/deleter.php?patient={{ pid|attr_url }}&csrf_token_form={{ csrf|attr_url }}"
+        href="../../../interface/patient_file/deleter.php?patient={{ pid|attr_url }}&csrf_token_form={{ csrf|attr_url }}"
```

Comment:
would use `{{ webroot|attr }}` instead of `{{ urlWebRoot|attr }}`

this is globally available in all twig templates via:
https://github.com/openemr/openemr/blob/master/src/Common/Twig/TwigExtension.php#L55